packages/domain/src/generators/react-component/files/__fileName__.component.spec.ts__tmpl__

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
What are the differences between your generator in terms of these template files and https://github.com/nrwl/nx/tree/master/packages/react/src/generators/component

Reply:
Ah yes, we could use that generator instead :) Simplify!